terra-jupyter-r/Dockerfile

```diff
@@ -7,9 +7,22 @@ COPY scripts $JUPYTER_HOME/scripts
 ENV TERRA_R_PLATFORM="terra-jupyter-r"
 ENV TERRA_R_PLATFORM_BINARY_VERSION=3.13.0
 
+# Install protobuf 3.16. Note this version comes from base deep learning image. Use `conda list` to see what's installed
+RUN cd /tmp \
```

Comment:
I've tried twice and this command fails for me.. does it work for you?
```
#7 1146.3 make[2]: Leaving directory '/tmp/protobuf-3.16.0/src'
#7 1146.3 make  check-TESTS
#7 1146.6 make[2]: Entering directory '/tmp/protobuf-3.16.0/src'
#7 1146.9 make[3]: Entering directory '/tmp/protobuf-3.16.0/src'
#7 1176.8 ../test-driver: line 109: 16923 Killed                  "$@" > $log_file 2>&1
#7 1176.8 FAIL: protobuf-test
#7 1177.0 PASS: protobuf-lazy-descriptor-test
#7 1177.0 PASS: protobuf-lite-test
#7 1177.7 PASS: google/protobuf/compiler/zip_output_unittest.sh
#7 1177.8 PASS: google/protobuf/io/gzip_stream_unittest.sh
#7 1177.8 PASS: protobuf-lite-arena-test
#7 1177.8 PASS: no-warning-test
#7 1177.8 ============================================================================
#7 1177.8 Testsuite summary for Protocol Buffers 3.16.0
#7 1177.8 ============================================================================
#7 1177.8 # TOTAL: 7
#7 1177.8 # PASS:  6
#7 1177.8 # SKIP:  0
#7 1177.8 # XFAIL: 0
#7 1177.8 # FAIL:  1
#7 1177.8 # XPASS: 0
#7 1177.8 # ERROR: 0
#7 1177.8 ============================================================================
#7 1177.8 See src/test-suite.log
#7 1177.8 Please report to protobuf@googlegroups.com
#7 1177.8 ============================================================================
#7 1177.8 make[3]: *** [test-suite.log] Error 1
#7 1177.8 Makefile:7980: recipe for target 'test-suite.log' failed
#7 1177.8 make[3]: Leaving directory '/tmp/protobuf-3.16.0/src'
#7 1177.9 make[2]: *** [check-TESTS] Error 2
#7 1177.9 Makefile:8086: recipe for target 'check-TESTS' failed
#7 1177.9 make[2]: Leaving directory '/tmp/protobuf-3.16.0/src'
#7 1177.9 make[1]: *** [check-am] Error 2
#7 1177.9 Makefile:8204: recipe for target 'check-am' failed
#7 1177.9 make[1]: Leaving directory '/tmp/protobuf-3.16.0/src'
#7 1177.9 make: *** [check-recursive] Error 1
#7 1177.9 Makefile:1881: recipe for target 'check-recursive' failed
------
executor failed running [/bin/sh -c cd /tmp   && wget https://github.com/protocolbuffers/protobuf/releases/download/v3.16.0/protobuf-all-3.16.0.tar.gz 	&& tar -xvzf protobuf-all-3.16.0.tar.gz 	&& cd protobuf-3.16.0/ 	&& ./configure 	&& make 	&& make check 	&& sudo make install 	&& sudo ldconfig 	%% rm -rf /tmp/protobuf-* 	&& cd ~]: exit code: 2
```

Reply:
it should be...one change I made after I tested locally is moving these to a `tmp` directory...I can try again locally

Reply:
Actually I'm trying again while giving my docker more memory

Reply:
Btw increasing my docker memory from 4G to 8G fixed this